org.kde.AwesomeBump.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "org.kde.AwesomeBump",
+    "branch": "Release",
```

Comment:
On flathub this has to be 'stable' - just leave it out of the manifest and it'll get set automatically.

Reply:
Hi, thank you for all the comments. TBH I was not sure how to correctly fill the manifest file. I will apply your suggestions.